src/test/java/com/lewisd/maven/lint/plugin/CheckMojoTest.java

```diff
@@ -1,128 +1,128 @@
-		Assert.assertEquals(expected, actual);
-	}
-
-	@Test
-	public void testErrorMessageForSummaryAndTwoReports() throws Exception {
-		final String expected = "[LINT] Violations found. For more details, see error messages above, or results in one of the following files: " + xmlFilePath + ", " + htmlFilePath;
-		final String actual = checkMojo.generateErrorMessage(Lists.newArrayList("summary", "xml", "html"));
-		Assert.assertEquals(expected, actual);
-	}
-
-	@Test
-	public void testErrorMessageForNoSummaryAndOneReport() throws Exception {
-		final String expected = "[LINT] Violations found. For more details see results in " + xmlFilePath;
-		final String actual = checkMojo.generateErrorMessage(Lists.newArrayList("xml"));
-		Assert.assertEquals(expected, actual);
-	}
-
-	@Test
-	public void testErrorMessageForNoSummaryAndTwoReports() throws Exception {
-		final String expected = "[LINT] Violations found. For more details see results in one of the following files: " + xmlFilePath + ", " + htmlFilePath;
-		final String actual = checkMojo.generateErrorMessage(Lists.newArrayList("xml", "html"));
-		Assert.assertEquals(expected, actual);
-	}
-
-	@Test
-	public void testErrorMessageForSummaryFileAndTwoReports() throws Exception {
-		final File summaryFile = new File("summary.txt");
-		final String summaryFilePath = summaryFile.getAbsolutePath();
-		setField(checkMojo, "summaryOutputFile", summaryFile);
-		final String expected = "[LINT] Violations found. For more details see results in one of the following files: " + summaryFilePath + ", " + xmlFilePath;
-		final String actual = checkMojo.generateErrorMessage(Lists.newArrayList("summary", "xml"));
-		Assert.assertEquals(expected, actual);
-	}
-
-
-	// TODO must use current version (currently not working with snapshot)
-	@Test
-	public void testDefaultProject() throws Exception {
-		String pom = "src/test/resources/bugs/issue-4-missing-pluginParameterEvaluation/pom.xml";
-		String goal = "com.lewisd:lint-maven-plugin:check";
-		File tempFile = new File(".").createTempFile("log", "log");
-		tempFile.deleteOnExit();
-		File targetDir = new File("src/test/resources/bugs/issue-4-missing-pluginParameterEvaluation/target");
-		targetDir.deleteOnExit();
-		// everything except the warnings are going into this log
-		String[] args = {goal, "-f", pom, "-l", tempFile.getPath(), "-e"};
-
-		PrintStream oldOut = System.out; // remember old System.out
-		try {
-			ByteArrayOutputStream stdout = new ByteArrayOutputStream();
-			System.setOut(new PrintStream(stdout));
-
-			int result = new MavenCli().doMain(args, ".", null, null);
-
-			if (result != 0) {
-				String log = FileUtils.readFileToString(tempFile);
-				fail("sth went wrong : " + log);
-			}
-
-			assertTrue("expected to have some warnings " + stdout.toString(), stdout.toString().contains("WARN "));
-		} finally {
-			System.setOut(oldOut); // cleanup
-		}
-	}
-}
+package com.lewisd.maven.lint.plugin;
```

Comment:
Any idea what happened here?  Github is showing that this whole file changed.  Was it wrong line-endings?